colin.spec

```diff
@@ -0,0 +1,74 @@
+%global pypi_name colin
+
+%if 0%{?rhel} && 0%{?rhel} <= 7
+%bcond_with python3
+%else
+%bcond_without python3
+%endif
+
+Name:           %{pypi_name}
+Version:        0.0.3
+Release:        1%{?dist}
+Summary:        Tool to check generic rules/best-practices for containers/images/dockerfiles.
+
+License:        GPLv3+
+URL:            https://github.com/user-cont/colin
+Source0:        https://files.pythonhosted.org/packages/source/c/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+
+%description
+`colin` is a tool to check generic rules/best-practices
+for containers/images/dockerfiles.
+
+%package -n     python3-%{pypi_name}
+Summary:        %{summary}
+%{?python_provide:%python_provide python3-%{pypi_name}}
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+Requires:       python3-docker
+Requires:       python3-requests
+Requires:       python3-pyxattr
+Requires:       conu
+Recommends:     atomic
+Recommends:     docker
+
+%description -n python3-%{pypi_name}
+`colin` as a tool to check generic rules/best-practices
+for containers/images/dockerfiles.
+
+%package -n %{pypi_name}-doc
+Summary:        colin documentation
+
+%description -n %{pypi_name}-doc
+Documentation for colin.
+
+%prep
+%autosetup -n %{pypi_name}-%{version}
+# Remove bundled egg-info
+rm -rf %{pypi_name}.egg-info
+
+%build
+%py3_build
+
+%install
+%py3_install
+
+%files -n python3-%{pypi_name}
+%license LICENSE
+%doc README.md
+%{_bindir}/%{pypi_name}
```

Comment:
I agree with @bkabrda; it would be great to have the executable in colin rpm

Reply:
So we should split the code into the following parts?

- `python3-colin` (core) (only basic checks)
- `colin` (only executable)
- additional checks+rulesets e.g. `python3-colin-redhat`

Does it mean also different pip packages and different repositories? Or is it only a 'problem' with `rpm` packages?

Reply:
python3-colin-redhat would be in private repo on our GitLab.

Reply:
Yes, of course. I've tried to make the package with additional checks/rulesets and it's really easy:
https://github.com/lachmanfrantisek/colin-openhouse/ (just WIP)

Reply:
how about `colin-ruleset-redhat`? I'm not sure whether we actually want to have two: one open (our images are open, nothing to hide, really), one closed (for checks which are tied to internal infra)